main.rkt

```diff
@@ -22,6 +25,9 @@
 
 ;; =============================================================================
 
+(define (square n)
+  (expt n 2))
+
```

Comment:
Also hilariously absent is `fact`. The question is how do we want to implement it..

```
(define (fact n)
  (let loop ([a 0] [n n])
    (if (= n 0)
        1
         (loop (* n a) (sub1 n)))))

(require math/special-functions)
(define fact (compose truncate gamma add1))
```
Also for naming do we prefer `fact` or `factorial`?